src/applications/gi/tests/03-vet-tec.e2e.spec.js

```diff
@@ -0,0 +1,59 @@
+const E2eHelpers = require('../../../platform/testing/e2e/helpers');
+const Timeouts = require('../../../platform/testing/e2e/timeouts');
+const GiHelpers = require('./gibct-helpers');
+const VetTecHelpers = require('./vet-tec-helpers');
+const vetTecProfile = require('./e2e/vet-tec-profile.json');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  const vetTecAttributes = vetTecProfile.data.attributes;
+
+  VetTecHelpers.initApplicationMock();
+
+  client.openUrl(`${E2eHelpers.baseUrl}/gi-bill-comparison-tool/`);
+
+  E2eHelpers.overrideSmoothScrolling(client);
+  client.timeoutsAsyncScript(2000);
+
+  // Landing Page
+  client
+    .waitForElementVisible('body', Timeouts.normal)
+    .waitForElementVisible('.gi-app', Timeouts.verySlow)
+    .axeCheck('.main');
+
+  VetTecHelpers.searchForVetTec(client);
+
+  // Search Page
+  GiHelpers.expectLocation(client, `/program-search`);
+  GiHelpers.selectFirstSearchResult(client);
+
+  // Profile Page
+  GiHelpers.expectLocation(
+    client,
+    `/profile/${vetTecAttributes.facility_code}/`,
+  );
+  client
+    .waitForElementVisible('.profile-page', Timeouts.normal)
+    .axeCheck('.main'); // commented out until 7727 is fixed
```

Comment:
Same comment about moving the axe check after the accordions have been opened.

Reply:
By default all accordions are expanded when loading a VET TEC profile page
see https://staging.va.gov/gi-bill-comparison-tool/profile/2V000105/HACK%20REACTOR%20SOFTWARE%20ENGINEERING%20ONLINE

Reply:
👍 Cool, thanks for confirming.